Makefile

```diff
@@ -35,7 +37,7 @@ $(TESTS):
 	# Build our own tests, using GTest
 	# TODO: Figure out how to name multiple CC and produce corresponding object for each
 	$(CXX) $(SRC_FLAGS) $(GTEST_FLAGS) $(PARSER_PATH)config_parser.cc server_config.cc $(TESTS).cc ${GTEST_DIR}/src/gtest_main.cc libgtest.a $(LDFLAGS) -o $(TESTS)
-	$(CXX) $(SRC_FLAGS) $(GTEST_FLAGS) request_handlers.cc request_handlers_test.cc ${GTEST_DIR}/src/gtest_main.cc libgtest.a $(LDFLAGS) -o request_handlers_test
+	$(CXX) $(SRC_FLAGS) $(GTEST_FLAGS) $(PARSER_PATH)config_parser.cc mime_types.cc server_config.cc request_router.cc request_handlers.cc request_handlers_test.cc ${GTEST_DIR}/src/gtest_main.cc libgtest.a $(LDFLAGS) -o request_handlers_test
```

Comment:
I was just thinking that something like the following might keep the style a little more consistent. Maybe not exactly the same since I have no idea how long the lines are below haha

```
$(CXX) $(SRC_FLAGS) $(GTEST_FLAGS) $(PARSER_PATH)config_parser.cc \
mime_types.cc server_config.cc request_router.cc request_handlers.cc \
request_handlers_test.cc ${GTEST_DIR}/src/gtest_main.cc libgtest.a \
$(LDFLAGS) -o request_handlers_test
```